### DISCUSSION TITLE
github organization space full - how to free up some space.

### DISCUSSION THREAD

**Discussion by bhavishyari** [community/community#31307] [Other Feature Feedback, Questions, & Ideas]
I have removed all the artifacts from all the repositories, cloned the repo to local, and checked the size of that repo, but the size of that repo is not more than 5MB. 

My GitHub organization left with 400MB in storage. Can someone tell me where should I look to free up some space from my GitHub account?

**Comment by jfullstackdev** [comment]
is that a public repo, so I can also take a look ? 

if not, try logging in in Incognito mode, maybe it's just cached data
